<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Backends:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="chBoxBReference">
         <property name="text">
          <string>Reference</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chBoxBResvg">
         <property name="text">
          <string>Resvg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chBoxBChrome">
         <property name="text">
          <string>Chrome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chBoxBFirefox">
         <property name="text">
          <string>Firefox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chBoxBSafari">
         <property name="text">
          <string>Safari</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chBoxBBatik">
         <property name="text">
          <string>Batik</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chBoxBInkscape">
         <property name="text">
          <string>Inkscape</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chBoxBLibrsvg">
         <property name="text">
          <string>librsvg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chBoxBQtSvg">
         <property name="text">
          <string>QtSvg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chBoxBLadybird">
         <property name="text">
          <string>Ladybird</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="chBoxShowTitle">
         <property name="text">
          <string>Show title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chBoxIndicateStatus">
         <property name="toolTip">
          <string>Draw a colored frame around a test image that indicates a test pass status.</string>
         </property>
         <property name="text">
          <string>Indicate status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chBoxShowDiff">
         <property name="toolTip">
          <string>Draw a colored frame around a test image that indicates a test pass status.</string>
         </property>
         <property name="text">
          <string>Show diff</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
